src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:35139/organizations/org3/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:35139/project/issues?id=sample&pullRequest=6294&issues=AWkKdp27HK0nbq4oj739&open=AWkKdp27HK0nbq4oj739)

[ISSUE_KEY]: AWkKdp27HK0nbq4oj739